src/resources/action-map/score.js

```diff
@@ -15,8 +16,12 @@ const scoreMapper = (map) => {
   return map.reduce((p, [k, v]) => p.concat(k, v), []);
 };
 
+const buildNestedQuery = (filter, ctx, target) => (filter === null ? {} : {
+  query: buildQuery(filter, ctx.allowedFields, target.substring(0, target.lastIndexOf('.')))
+});
```

Comment:
Does it not work correctly if you use match all here instead of `{}`?

Reply:
It's the same behavior, according to here: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-function-score-query.html

> If no filter is given with a function this is equivalent to specifying "match_all": {}

Reply:
My preference would be to do that since it aligns with the ordering query. It would also remove object assign. Thoughts?